### PR TITLE
auto remove in docker

### PR DESCRIPTION
**Pull request by licode**

### CODE REVIEW

File: scripts/nsls2-dev-build.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ docker pull $IMAGE_NAME
 
 echo "Running the docker container"
 
-cat << EOF | docker run -i \
+cat << EOF | docker run -i --rm\
```
Comment (mrakitin) [inline scripts/nsls2-dev-build.sh]: I would add an extra blank space just in case. Same for the below.

Comment (licode) [reply scripts/nsls2-dev-build.sh]: good catch.

**Comment by mrakitin** [comment]
Looks great to me, thanks @licode!

**Comment by licode** [comment]
thanks a lot!

**Pull request merged**